packages/hardhat-zksync-solc/src/utils.ts

```diff
@@ -17,7 +17,8 @@ export async function getZksolcPath(version: string): Promise<string> {
 }
 
 export function getZksolcUrl(version: string): string {
-    const platform = process.platform;
+    const platform = process.platform == 'darwin' ? 'macosx' : process.platform;
```

Comment:
Wouldn't it be right to use `darwin` for binaries in the repo though?

Reply:
I have no idea what is considered "right" here. Why `amd64` and not `x64`? Why include `-musl` only for linux? I believe they are naming this in the same way as [`solc-bin`](https://github.com/ethereum/solc-bin) does, but I don't see any logic behind it. I'm happy to change the code here or rename the binaries, whatever makes more sense.